src/App.tsx

```diff
@@ -1,10 +1,50 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useMemo, useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Header } from './component/Header/Header';
+import { Footer } from './component/Footer/Footer';
+import { TodoApp } from './component/TodoApp/TodoApp';
+import { getTodos } from './api/todos';
+import { TodoFilter } from './types/TodoFilter';
 
-const USER_ID = 0;
+const USER_ID = 10610;
+const timeDelay = 3000;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filteredBy, setFilteredBy] = useState<TodoFilter>(TodoFilter.ALL);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosFromServer) => {
+        setTodos(todosFromServer);
+      })
+      .catch(() => {
+        setIsError(true);
+        setTimeout(() => {
```

Comment:
create useEffect for this logic, check if(isError) and clear timeout in unmount effect